draft-pauly-httpbis-geoip-hint.md

```diff
@@ -73,63 +95,99 @@ For example, the header for an entry "192.0.2.5,US,US-AL,Alabaster" would be:
     Sec-CH-IP-Geo = "US,US-AL,Alabaster"
 ~~~
 
-This field also defines a parameter, "feed", that contains the URI of the
-IP geolocation feed that is authoritative for this entry. For example:
-
-~~~
-    Sec-CH-IP-Geo = "SG,SG-01,Singapore"; feed="https://noc.ietf.org/geo/google.csv"
-~~~
-
-Servers that can provide different content based on Geohash hints SHOULD include
-the headers in their "Accept-CH" list.
+Given that the Sec-CH-IP-Geo is a high-entropy client hint, meaning it is
+a client hint that is not in the low-entropy hint table, the server
+MUST explicitly opt-in to receive the Geo Client Hint as defined in
+{{?RFC8942}}. It will not be sent by default and the server MAY
```

Comment:
The `?` means informative reference, but we want a normative reference. Those are represented with `!` but that's already done on line 43, so just reference without either `?` nor `!`

```suggestion
{{RFC8942}}. It will not be sent by default and the server MAY
```